azurerm/resource_arm_application_gateway.go

```diff
@@ -50,6 +50,40 @@ func resourceArmApplicationGateway() *schema.Resource {
 				ForceNew: true,
 			},
 
+			"identity": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Computed: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"type": {
+							Type:             schema.TypeString,
+							Required:         true,
+							DiffSuppressFunc: ignoreCaseDiffSuppressFunc,
```

Comment:
all new fields in the provider are case-sensitive, as such can we remove this?

```suggestion
```

Reply:
OK